### ISSUE TITLE
Default `role_name` specified by `target_role_name` of base account

### ISSUE THREAD

**Issue by tilfin** [tilfin/aws-extend-switch-roles#91]
For example,
```
[default]
role_name = something

[foo]
aws_account_id = 000011112222

[bar]
aws_account_id = 000011113333
```
is equals to
```
[foo]
aws_account_id = 000011112222
role_name = something

[bar]
aws_account_id = 000011113333
role_name = something
```
, it is useful when a person has a role with the same name for each AWS account.

**Comment by tilfin** [comment]
A profile with the `role_name` cannot be distinguished whether it is a source one. 
If it's `target_role_name`, it is recognized as assuming the role name.

**Comment by tilfin** [comment]
released on v0.13.0
